<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="29956" started-at="2020-05-01T01:57:06Z" finished-at="2020-05-01T01:57:36Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="29956" started-at="2020-05-01T01:57:06Z" finished-at="2020-05-01T01:57:36Z">
      <class name="com.qa.testcase.Testcreatecustomer">
        <test-method status="PASS" signature="pretest()[pri:0, instance:com.qa.testcase.Testcreatecustomer@5622fdf]" name="pretest" is-config="true" duration-ms="17929" started-at="2020-05-01T01:57:06Z" finished-at="2020-05-01T01:57:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pretest -->
        <test-method status="FAIL" signature="testcreatecustomer()[pri:0, instance:com.qa.testcase.Testcreatecustomer@5622fdf]" name="testcreatecustomer" duration-ms="11893" started-at="2020-05-01T01:57:24Z" finished-at="2020-05-01T01:57:36Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.pages.Taskpage.addnew(Taskpage.java:25)
at com.qa.testcase.Testcreatecustomer.testcreatecustomer(Testcreatecustomer.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcreatecustomer -->
      </class> <!-- com.qa.testcase.Testcreatecustomer -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
